backend/src/packages/chat-messages/chat-message.repository.ts

```diff
@@ -60,7 +60,10 @@ class ChatMessageRepository implements Repository {
   }
 
   public delete(): ReturnType<Repository['delete']> {
-    return Promise.resolve(true);
+    //TODO
+    const deletedId = 0;
```

Comment:
should be deletedCount, rename it everywhere
```suggestion
    const deletedCount = 0;
```

Reply:
done